app/views/manuals/index.html+new_navigation.erb

```diff
@@ -0,0 +1,15 @@
+<%=
+  render(
+    'content_for_manual',
+    presented_manual: presented_manual,
+    title: presented_manual.full_title
+  )
+%>
+<%= content_for :extra_head_content do %>
+  <%= ab_test.requested_variant.analytics_meta_tag.html_safe %>
+<% end %>
+<%= render 'govuk_component/beta_label' %>
+<%= render 'govuk_component/breadcrumbs',
+  breadcrumbs: presented_manual.taxonomy_breadcrumbs %>
+<%= render 'header', presented_manual: presented_manual %>
+<%= render 'manual', presented_manual: presented_manual %>
```

Comment:
Ah, I see now why we extracted `manual`, `manual_section`, and `manual_updates` it's so we can use them in these new a/b test commits.  Ignore my previous comment I suppose.  I reckon it might be good to flatten them down when/if we decide to keep the new navigation behaviour.

I've a feeling however that folk are going to want (particularly on the HMRC manuals which can be deeply nested as manual -> section -> section -> section) the manual breadcrumbs to be closer to the content (e.g. after the header).